src/Api/Controllers/OrganizationsController.cs

```diff
@@ -190,7 +190,7 @@ public async Task<OrganizationResponseModel> PostLicense(OrganizationCreateLicen
             }
 
             var policies = await _policyRepository.GetManyByUserIdAsync(user.Id);
-            if (policies.Any(policy => policy.Type == PolicyType.SingleOrg))
+            if (policies.Where(policy => policy.Enabled).Any(policy => policy.Type == PolicyType.SingleOrg))
```

Comment:
Same, combine these (reduces compute cycles in `IEnumerable<T>` extension methods/iterators)